tools/oesign/oesign.c

```diff
@@ -174,6 +178,50 @@ static int _load_config_file(const char* path, ConfigFileOptions* options)
 
             options->security_version = n;
         }
+        else if (strcmp(str_ptr(&lhs), "IsvFamilyID_H") == 0)
```

Comment:
This is not a good design for user-facing input configuration. Intel's initial design for this is awkward and we would not want developers to have to think about this value specified in the configuration as a pair of uint64 decimal numbers instead of just as a single 128-bit UUID.

We should restructure ISV Family ID and Extended Product ID to take a standard UUID/GUID formatted or hexadecimal strings instead.

Reply:
Will update it with oe_uuid_t instead